### PR TITLE
WRO-4784: Added A11y feature for editable scroller

### PR DESCRIPTION
**Pull request by jeonghee27**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Need to support A11y feature on for editable scroller

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Specs : 
* When focus item in Scroller (I fixed sampler. It should be implemented on App)
<item label> + <"Edit mode to press and hold OK key")
* When entering edit mode and focusing for the first time
<item label> + <"Move left and right or press up key to delete")
*  press left / right
<item label> + <"moved to the left of"> + <item label.position+1>
<item label> + <"moved to the right of"> + <item label.position-1>
App should add `data-item-label` on the each item's container
* press left/right when focus on leftmost/rightmost
<"Leftmost">
<"Rightmost">
* (edit mode) press up  (I fixed sampler. It should be implemented on App)
<"Delete"> + <"button">
* done (OK key in edit mode)
<item label> + <"move complete">
=> Since the focus element is read again after re-rendering, it is implemented to call `annouce` after a arbitrary 300ms.
There is other way to modify the label twice in the app, But I thought this was not good way to guide.  So I just used setTimout.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-4784

### Comments
Enact-DCO-1.0-Signed-off-by: Jeonghee Ahn (jeonghee27.ahn@lge.com)

### CODE REVIEW

**Review by seunghoh — COMMENTED** [review]
hm.. I have some considerations
1. Do we really need `data-item-label`? It works as public prop but too specific cases
2. What if we just read "left/right" then, we probably no need to use `data-item-label`

One more thing is that I don't see the sample code when the item is deleted.

**Comment by jeonghee27** [comment]
>hm.. I have some considerations
>1. Do we really need data-item-label? It works as public prop but too specific cases
>2. What if we just read "left/right" then, we probably no need to use data-item-label
>One more thing is that I don't see the sample code when the item is deleted.

1. I changed to use `aria-label` instead of `data-item-label`.
2. It is inevitable to receive a `label` from the outermost DOM of each item, because it is not known what structure the actual item will be composed of or whether the textContext will always exist.
3. "Item deleted" spec has been removed

**Review by seunghoh — APPROVED** [review]
LGTM

File: Scroller/EditableWrapper.js [hunk 1]
```diff
@@ -351,6 +406,13 @@ const EditableWrapper = (props) => {
 					const orders = finalizeOrders();
 					forwardCustom('onComplete', () => ({orders}))({}, editable);
 					reset();
+
+					setTimeout(() => {
+						announceRef.current.announce(
+							selectedItemLabel + $L('move complete'),
+							true
+						);
+					}, 300); // An arbitrary delay at a level that is not ignored by the new focus element
```
Comment (0x64) [inline Scroller/EditableWrapper.js:415]: Let's make a constant for this delay since it could be updated later if requested.

**Pull request merged**